src/mono/wasm/Wasm.Build.Tests/Templates/WasmTemplateTestsBase.cs

```diff
@@ -186,6 +198,23 @@ public virtual (string projectDir, string buildOutput) BuildProject(
             return (_projectDir, res.Output);
         }
 
+        if (EnvironmentVariables.UseJavascriptBundler && buildOptions.IsPublish)
+        {
+            string publicWwwrootDir = Path.GetFullPath(Path.Combine(GetBinFrameworkDir(configuration, forPublish: true), ".."));
+            File.Copy(Path.Combine(BuildEnvironment.TestAssetsPath, "JavascriptBundlers", "package.json"), Path.Combine(publicWwwrootDir, "package.json"));
+            File.Copy(Path.Combine(BuildEnvironment.TestAssetsPath, "JavascriptBundlers", "rollup.config.mjs"), Path.Combine(publicWwwrootDir, "rollup.config.mjs"));
+
+            string npmPath = s_isWindows ? @"C:\Program Files\nodejs\npm.cmd" : "/bin/npm";
```

Comment:
we can't hardcode the path like this but I guess we can rely on `npm.cmd` / `npm` being on PATH